gossip/src/crds.rs

```diff
@@ -746,6 +767,10 @@ impl CrdsStats {
             GossipRoute::PullResponse => self.pull.record_fail(entry),
         }
     }
+
+    fn record_redundant_pull(&mut self) {
+        self.redundant_pull += 1;
+    }
```

Comment:
feels a bit unnecessary to add a fn for a one-liner which is only used one place.

Reply:
ok i can just increment directly